src/screens/Notifications/ConfirmEmailScreen.tsx

```diff
@@ -189,16 +188,19 @@ class ConfirmEmailScreen extends Component<Props, State> {
         const hashes = await Promise.all(walletsToSubscribe!.map(wallet => getWalletHashedPublicKeys(wallet)));
         this.props.unsubscribe(hashes, email);
       },
-      onCodeConfirm: () =>
-        CreateMessage({
-          title: i18n.message.success,
-          description: i18n.notifications.updateNotificationPreferences,
-          type: MessageType.success,
-          buttonProps: {
-            title: i18n.message.goToWalletDetails,
-            onPress: () => (onBack ? onBack() : navigation.popToTop()),
-          },
-        }),
+      onCodeConfirm: () => {
+        onSuccess
```

Comment:
please add a comment this is to refactor and brief sentence what's need to be refactored